docs/mobile/events.md

```diff
@@ -21,46 +20,57 @@ application lifecycle. This event is meant to be used in conjunction with
 `os.name` [resource semantic convention](/docs/resource/os.md) to identify the
 mobile operating system (e.g. Android, iOS).
 
-### iOS
+Payload attributes MUST be used to describe the state of the application at the
+time of the event. The following table describes the payload attributes that MUST
+be used to describe the state of the application at the time of the event.
 
-<!-- semconv ios.lifecycle.events -->
-The event name MUST be `device.app.lifecycle`.
-
-| Attribute  | Type | Description  | Examples  | Requirement Level |
-|---|---|---|---|---|
-| `ios.state` | string | This attribute represents the state the application has transitioned into at the occurrence of the event. [1] | `active` | Required |
+The `android.state` and `ios.state` attributes are mutually exclusive and MUST
+NOT be used together, each attribute MUST be used with it's corresponding
+`os.name` [resource semantic convention](/docs/resource/os.md) value. For
+example, if the `os.name` attribute is set to `android` then the
+`android.state` attribute MUST be used and the `ios.state` attribute MUST NOT
+be used. If the `os.name` attribute is set to `ios` then the `ios.state`
+attribute MUST be used and the `android.state` attribute MUST NOT be used.
```

Comment:
I don't think the "For example...." part makes it any clearer (a little rambly to me). I think just saying that exactly one of `ios.state` or `android.state` must be present.

Reply:
I'm also wondering whether we need 2 separate "*.state" attrs each prefixed with the same value that is also present in the attr `os.name`.

Reply:
I'm going to leave the 2 states for this PR and that can be a follow up one, I have removed for example as @lmolkova suggestion of the requirement level I think makes it clearer.